test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -317,7 +317,7 @@ u256 EVMInstructionInterpreter::eval(
 		logTrace(_instruction, arg);
 		return (0xcccccc + arg[1]) & u256("0xffffffffffffffffffffffffffffffffffffffff");
 	case Instruction::CREATE2:
-		accessMemory(arg[2], arg[3]);
+		accessMemory(arg[1], arg[2]);
 		logTrace(_instruction, arg);
 		return (0xdddddd + arg[1]) & u256("0xffffffffffffffffffffffffffffffffffffffff");
```

Comment:
Shouldn't this use `arg[3]` aka salt? Or a both `arg[1] + arg[3]` for added resilience?

Reply:
Since the optimizer does not affect salt, status quo should be fine. Actually, the problem is zero-length ranges that will be addressed in an upcoming PR.